src/coreimage.cs

```diff
@@ -4729,70 +4730,75 @@ interface CIAttributedTextImageGenerator {
 	[TV (11,0)]
 	[BaseType (typeof (CIFilter))]
 	interface CIBarcodeGenerator {
-		// TODO: Needs review
-		//[CoreImageProperty ("inputBarcodeDescriptor")]
-		//CIBarcodeDescriptor BarcodeDescriptor { get; set; }
+		[CoreImageFilterProperty ("inputBarcodeDescriptor")]
+		CIBarcodeDescriptor BarcodeDescriptor { get; set; }
 	}
 
 	[CoreImageFilter]
 	[iOS (11,0)]
 	[Mac (10,13)]
 	[TV (11,0)]
+	// Maybe 'typeof (CIFilter)' (shared 'Scale' and 'AspectRatio' property).
```

Comment:
not sure I understand the comment above ? the base type is already CIFilter

Reply:
Wrong copy and paste (: